shell/apps/vr/index.html

```diff
@@ -3,55 +3,56 @@
 <head>
 
   <title>Arcs</title>
+
   <meta charset="utf-8">
-  <link rel="manifest" href="manifest.json"/>
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
-  <link rel="shortcut icon" href="../logo_64.png">
-  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 
-  <style>
-    body {
-      font-family: 'Roboto', sans-serif;
-      margin: 0;
-      overflow-x: hidden;
-      overflow-y: scroll;
-    }
-    @font-face {
-      font-family: 'Material Icons';
-      font-style: normal;
-      font-weight: 400;
-      src: url(https://fonts.gstatic.com/s/materialicons/v30/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
-    }
-  </style>
+  <link rel="manifest" href="../common/manifest.json"/>
+  <link rel="shortcut icon" href="../common/logo_64.png">
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Google+Sans">
 
   <script>
-    window.shellPath = "../..";
+    (() => {
+      const debug = (new URL(document.location)).searchParams.has('debug');
```

Comment:
👍

This seems more maintainable to me- just one index with a flag. Harder to discover though but we need the URL params doc anyways.

Reply:
Maintaining two variants of `index.html` was becoming a burden, that's what drove this change. 

I don't have a sense re: discoverability. 

The debug mode is not that great anyway, it's only useful for testing runtime changes that affect Shell. We need to employ one of the solutions for feeding modules to the workers to really get a build-free mode.

Reply:
FTR there is increased activity recently on the modules-in-workers bug, but it's probably still a ways out. http://crbug.com/680046